src/SamplesApp/UITests.Shared/Windows_System/MemoryManagerTests.xaml

```diff
@@ -0,0 +1,24 @@
+﻿<Page
+    x:Class="UITests.Shared.Windows_System.MemoryManagerTests"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:UITests.Windows_System.MemoryManager"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:xamarin="http://uno.ui/xamarin"
+	mc:Ignorable="d xamarin">
+
+	<xamarin:Grid>
+		<Grid.RowDefinitions>
+			<RowDefinition Height="Auto" />
+			<RowDefinition Height="*" />
+		</Grid.RowDefinitions>
+
+		<StackPanel Background="Honeydew">
+			<TextBlock x:Name="ValueMemoryManager" Text="Click to display AppMemoryUsage and AppMemoryUsageLimit values"/>
```

Comment:
Let's use individual fields, it's easier to deal with when working with [UI Tests](https://platform.uno/docs/articles/contributing/guidelines/creating-tests.html).

```suggestion
			<TextBlock x:Name="AppMemoryUsageResult" Text="N/A"/>
			<TextBlock x:Name="AppMemoryUsageLimitResult" Text="N/A"/>
```

You don't need a UI test for now, but can you add a runtime test that validates that the values are not zero, for instance ? You can add it here: https://github.com/unoplatform/uno/tree/master/src/Uno.UI.RuntimeTests/Tests/Windows_System